preview/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default class Preview extends React.Component {
+
+  render() {
+    return (
+      <div>
+        abcd
```

Comment:
Is this test data? :D

Reply:
Nope :) It's prepared for preview page of components